### DISCUSSION TITLE
Tarteaucitron se charge uniquement lorsque l'on est connecté à la console d'administration de WordPress

### DISCUSSION THREAD

**Discussion by Eliophot** [AmauriC/tarteaucitron.js#759] [Q&A]
Bonjour,
Sur ce site : https://www.moulindulandion.com
On utilise le plugin WordPress + compte premium.
Lorsque l'on est connecté, TAC fonctionne. Mais dans le cas contraire, TAC n'est jamais exécuté.
Pourtant on peut voir dans le code source que le js est présent (load.js)
Nous utilisons WP Rocket pour la gestion du cache, mais même quand on désactive celui-ci, impossible de charger TAC.
Auriez-vous une idée du pourquoi et du comment ?
D'autant que nous utilisons cette solution sur une dizaine d'autres sites sans soucis...
Merci

**Comment by AmauriC** [comment]
Hello,
Il faut exclure `https://tarteaucitron.io/load.js` des options de minification/cache de wprocket ; )

**Comment by Eliophot** [comment]
Merci, je viens de le faire (on peut le voir dans le code source), mais ça ne marche toujours pas.
![image](https://user-images.githubusercontent.com/81296123/124948643-284af900-e011-11eb-8cc8-5001cfc5fd73.png)
Il doit y avoir une spécificité sur ce site, mais alors quoi ?

**Reply by AmauriC** [reply]
J'ai toujours la version 'wprocket' : /
![Capture d’écran 2021-07-08 à 17 28 40](https://user-images.githubusercontent.com/8764074/124949610-0140f700-e012-11eb-95ce-2b0bf674940b.png)

**Reply by Eliophot** [reply]
Effectivement... quelle est cette magie ? en fait en ajoutant :  https://tarteaucitron.io/load.js dans les fichiers à exclure ça ne marche pas. Bon j'ai décoché la minification et on est bon. Merci bien ;)
